### DISCUSSION TITLE
How to use QEMU

### DISCUSSION THREAD

**Discussion by Schlumpf** [OE4T/tegra-demo-distro#169] [Q&A] [answered]
Sorry for this newbie question:
Is it possible to test my Jetson Xavier NX software with QEMU?

**Comment by madisongh** [comment, marked as answer]
You can use it for generic binaries, but anything that calls on Jetson-specific libraries (i.e., those that make use of on-chip hardware), CUDA, etc. can't be run in QEMU.

**Reply by Schlumpf** [reply]
OK, I was already thinking something like that.
So I have to create an image without CUDA stuff, select `tegrademo` as distro and select `qemuarm` as machine?

**Reply by madisongh** [reply]
Yep, although `qemuarm64` would be a better choice.

**Reply by Schlumpf** [reply]
Many thanks for the explanation

**Comment by Deepak-Beniya** [comment]
after selecting the machine as qemux86-64 and images are generated.
Then how to start the virualization ?
